test/language/expressions/arrow-function/static-init-await-reference.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2021 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-identifiers
```

Comment:
As of https://github.com/tc39/proposal-class-static-block/pull/46, there are no longer any changes to `sec-identifiers`, these tests will likely need to be moved to `sec-class-definitions` (due to the change to use `+Await`), or to `sec-class-definitions-static-semantics-early-errors` (due to the new early error when `ContainsAwait` is **true**).